whispia-auth/src/main/kotlin/com/whispia/auth/jwt/JwtManagerImpl.kt

```diff
@@ -0,0 +1,113 @@
+        Decoders.BASE64.decode(jwtProperties.secret)
+    )
+    override fun generateToken(email: String, authorities: Collection<String>): TokenInfo {
+        log.info("토큰 생성: email={} authorities={}", email, authorities.joinToString(","))
+        val now = Date()
+        val accessTokenExpiresIn = Date(now.time + jwtProperties.accessTokenValidityInSeconds * 1000)
+        val refreshTokenExpiresIn = Date(now.time + jwtProperties.refreshTokenValidityInSeconds * 1000)
+
+        val authoritiesString = authorities.joinToString(",")
+
+        // Access Token 생성
+        val accessToken = Jwts.builder()
+            .subject(email)
+            .claim("auth", authoritiesString)
+            .issuedAt(now)
+            .expiration(accessTokenExpiresIn)
+            .signWith(key)
+            .compact()
+
+        // Refresh Token 생성.
+        val refreshToken = Jwts.builder()
+            .subject(email)
+            .expiration(refreshTokenExpiresIn)
+            .signWith(key)
+            .compact()
+
+        log.debug("액세스 토큰 생성: {}, 리프레시 토큰 생성: {}", accessToken, refreshToken)
+        return TokenInfo("Bearer", accessToken, refreshToken)
+    }
+
+    override fun validateToken(token: String): Boolean {
+        log.debug("토큰 검증: {}", token) // 앞 10자리만 로그
+        return try {
+            parseClaims(token)
+            true
+        } catch (e: SecurityException) {
+            throw AuthorizationException(ErrorCode.UNAUTHORIZED)
+        } catch (e: SignatureException) {
+            throw AuthorizationException(ErrorCode.UNAUTHORIZED)
+        }catch (e: MalformedJwtException) {
+            throw AuthorizationException(ErrorCode.JWT_MALFORMED)
+        } catch (e: ExpiredJwtException) {
+            throw AuthorizationException(ErrorCode.JWT_EXPIRED)
+        } catch (e: UnsupportedJwtException) {
+            throw AuthorizationException(ErrorCode.JWT_INVALID)
+        } catch (e: IllegalArgumentException) {
+            throw AuthorizationException(ErrorCode.JWT_MISSING)
+        }
+    }
+
+    override fun getAuthentication(token: String): Authentication {
+        val claims = parseClaims(token)
+        val authorities = claims["auth"]?.toString()?.split(",")
+            ?.map { SimpleGrantedAuthority(it) }
+            ?: emptyList()
+
+        log.debug("Authentication 객체 추출: email={} authorities={}", claims.subject, authorities.map { it.authority })
+        val principal = User(claims.subject, "", authorities)
+
+        return UsernamePasswordAuthenticationToken(principal, "", authorities)
+    }
+
+    override fun resolveToken(request: HttpServletRequest): String? {
+        val bearerToken = request.getHeader("Authorization")
+        log.debug("Authorization 헤더에서 토큰 확인, present={}", StringUtils.hasText(bearerToken))
+
+        return if (StringUtils.hasText(bearerToken) && bearerToken.startsWith("Bearer ")) {
+            bearerToken.substring(7)
+        } else {
+            null
+        }
```

Comment:
예외 안던지고 null 반환해도 괜찮을까요?
```suggestion
        if (!StringUtils.hasText(bearerToken) && !bearerToken.startsWith("Bearer ")) {
            throw AuthorizationException(ErrorCode.JWT_INVALID)
        }
        
        return bearerToken.substring(7)
```

이런 느낌을 생각해봤어요